P3D/Core/GameController.vb

```diff
@@ -81,6 +81,13 @@ Public Class GameController
             Security.HackerAlerts.Activate()
         End If
 
+        ' This is a nice workaround for turkish users because it is already annoying enough to force them to change system language...
+        If CultureInfo.CurrentCulture.EnglishName = "Turkish" Then
```

Comment:
Why do we change only the Turkish language parsing? Are other languages fine?
Couldn't this be achieved by using an inveriant parsing?
The specific issue is not stated, but I assume this is something about us showing or parsing in wrong format?
Please specify!